hack/metrodr-setup.sh

```diff
@@ -0,0 +1,229 @@
+#!/bin/bash
+
+export MANAGEDCLUSTER1="${MANAGEDCLUSTER1:-cluster1}"
+export MANAGEDCLUSTER2="${MANAGEDCLUSTER2:-managedCluster2}"
+export HUBCLUSTER="${HUBCLUSTER:-hub}"
+export CEPHCLUSTER="${CEPHCLUSTER:-cephcluster}"
+export MANAGEDCLUSTER_ROOK_NAMESPACE="${MANAGEDCLUSTER_ROOK_NAMESPACE:-rook-ceph}"
+basedir="$(dirname "$(realpath "$0")")"
+ramen_hack_directory_path_name=$(dirname $0)
+source $ramen_hack_directory_path_name/ocm-minikube.sh
+
+export PATH=${HOME}/.local/bin:${PATH}
+
+function metrodr_minikube_start() {
+        minikube_start "${HUBCLUSTER}"
+	minikube_start "${MANAGEDCLUSTER1}"
+	minikube_start "${CEPHCLUSTER}"
+}
+
+function metrodr_external_ceph_cluster_create() {
+        kubectl --context "${CEPHCLUSTER}" create -f https://raw.githubusercontent.com/rook/rook/master/deploy/examples/common.yaml
+        kubectl --context "${CEPHCLUSTER}" create -f https://raw.githubusercontent.com/rook/rook/master/deploy/examples/crds.yaml
+        kubectl --context "${CEPHCLUSTER}" create -f https://raw.githubusercontent.com/rook/rook/master/deploy/examples/operator.yaml
+	#echo "======Deploying cluster=======" && sleep 100
+	kubectl --context "${CEPHCLUSTER}" create -f "${basedir}/dev-rook-cluster.yaml"
+	#echo "======Deploying toolbox=======" && sleep 100
+        kubectl --context "${CEPHCLUSTER}" create -f https://raw.githubusercontent.com/rook/rook/master/deploy/examples/toolbox.yaml
+	#echo "======Deploying pool=======" && sleep 100
+        kubectl --context "${CEPHCLUSTER}" create -f "${basedir}/dev-rook-rbdpool.yaml"
+	# echo "======Deploying filesystem=======" && sleep 100
+        # kubectl --context ${CEPHCLUSTER} create -f https://raw.githubusercontent.com/rook/rook/master/deploy/examples/filesystem.yaml
+	#echo "======Deploying rbd storageclass=======" && sleep 100
+        kubectl --context "${CEPHCLUSTER}" create -f "${basedir}/dev-rook-sc.yaml"
+	# echo "======Deploying fs storageclass=======" && sleep 100
+        # kubectl --context ${CEPHCLUSTER} create -f https://raw.githubusercontent.com/rook/rook/master/deploy/examples/csi/cephfs/storageclass.yaml
+
+	#echo "======Patching default storageClass to rbd"
+        kubectl --context "${CEPHCLUSTER}" patch storageclass rook-ceph-block -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
+        kubectl --context "${CEPHCLUSTER}" patch storageclass standard -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}'
+        kubectl --context "${CEPHCLUSTER}" get storageclass
+}
+
+function check_for_command() {
+	if ! command -v "$1" >/dev/null
+	then
+		echo "command $1 not found"
+		exit 1
+	fi
+}
+
+function externalCluster_setFSID() {
+        declare rook_tools_pod
+        rook_tools_pod=$(kubectl --context "$1" -n "$2" get pods -o name | grep tools)
```

Comment:
Using $1 and $2 make this code very hard to understand. Please do not add this in new code. This should be eliminated entirely in this project.